tests/php/Notification/NotifierTest.php

```diff
@@ -208,7 +213,8 @@ public function testPrepareOne2One(string $uid, string $displayName, string $par
 					'type' => 'call',
 					'id' => 1234,
 					'name' => $displayName,
-					'call-type' => 'one2one'
+					'call-type' => 'one2one',
+					'icon-url' => '',
```

Comment:
Please instead fix the test so it expects that the result of the avatar service is returned here

Reply:
I  looked the unit test and I think that the integration tests at this case is best because to reproduce this at unit tests we will need to mock a lot of methods, including the string of avatar URL.